submissions/IrynaY/react-friends-app/src/services/index.js

```
@@ -0,0 +1,8 @@
+const URL = 'https://rickandmortyapi.com/api/character/'
+
+export const fetchCharacterList = (query) => {
+  let uri = URL
+  if(query)
+    uri += '?' + query
+  return fetch(uri).then(resp => !resp.ok ? [] : resp.json())
```

If `!resp.ok` you should throw an error :)

---

dirty hack fail ))